### PR TITLE
add database

### PR DESCRIPTION
**Pull request by LaserHuy**

### CODE REVIEW

**Review by yazidsef — CHANGES_REQUESTED** [review]
dinwi on  a parlé de name et firstname et adress de mettre name first name dans nom-complet et supp directement adress

File: database.sql [hunk 1]
```diff
@@ -61,3 +61,127 @@ ALTER TABLE `item`
...
+  `firstname` VARCHAR(100),
+  `name` VARCHAR(150),
+  `email` VARCHAR(150),
+  `tel` INT(10),
+  `degree` VARCHAR(100),
+  `birthday` DATE,
+  `address` TEXT,
+  `avatar_image` VARCHAR(255),
+  PRIMARY KEY (`id`)
+) ENGINE=InnoDB DEFAULT CHARSET=latin1;
```
Comment (Benja-san) [inline database.sql:80]: CHARSET to UTF8 (generalCI or AI CI)

File: database.sql [hunk 2]
```diff
@@ -61,3 +61,127 @@ ALTER TABLE `item`
...
+  `location` VARCHAR(100),
+  `date` DATE,
+  `duration` VARCHAR(50),
+  `degree` VARCHAR(100),
+  `financing_supported` BOOLEAN,
+  `discipline_id` INT,
+  `url_image` VARCHAR(255),
+  PRIMARY KEY (`id`),
+  FOREIGN KEY (`discipline_id`) REFERENCES `Discipline`(`id`)
+) ENGINE=InnoDB DEFAULT CHARSET=latin1;
```
Comment (Benja-san) [inline database.sql:100]: CHARSET to UTF8 (generalCI or AI CI)

File: database.sql [hunk 3]
```diff
@@ -61,3 +61,127 @@ ALTER TABLE `item`
...
 /*!40101 SET CHARACTER_SET_RESULTS=@OLD_CHARACTER_SET_RESULTS */;
 /*!40101 SET COLLATION_CONNECTION=@OLD_COLLATION_CONNECTION */;
+
+--
+-- Structure de la table `Student`
+--
+
+CREATE TABLE `Student` (
+  `id` INT NOT NULL,
+  `firstname` VARCHAR(100),
```
Comment (Benja-san) [inline database.sql:71]: can firstname be null ?

File: database.sql [hunk 4]
```diff
@@ -61,3 +61,127 @@ ALTER TABLE `item`
...
+  `name` VARCHAR(150),
```
Comment (Benja-san) [inline database.sql:72]: can this field be null ?

File: database.sql [hunk 5]
```diff
@@ -61,3 +61,127 @@ ALTER TABLE `item`
...
+  `email` VARCHAR(150),
```
Comment (Benja-san) [inline database.sql:73]: can this field be null ?

File: database.sql [hunk 6]
```diff
@@ -61,3 +61,127 @@ ALTER TABLE `item`
...
+  PRIMARY KEY (`id`)
+) ENGINE=InnoDB DEFAULT CHARSET=latin1;
+
+--
+-- Structure de la table `Course`
+--
+
+CREATE TABLE `Course` (
+  `id` INT NOT NULL,
+  `name` VARCHAR(100),
```
Comment (Benja-san) [inline database.sql:88]: can this field be null ?

File: database.sql [hunk 7]
```diff
@@ -61,3 +61,127 @@ ALTER TABLE `item`
...
+  `description` VARCHAR(150),
```
Comment (Benja-san) [inline database.sql:89]: can this field be null ? and are 150 characters for a description enough ?

File: database.sql [hunk 8]
```diff
@@ -61,3 +61,127 @@ ALTER TABLE `item`
...
+  `capacity` VARCHAR(150),
```
Comment (Benja-san) [inline database.sql:90]: integer better than varchar here

File: database.sql [hunk 9]
```diff
@@ -61,3 +61,127 @@ ALTER TABLE `item`
...
+-- Structure de la table `Les_eleves_d_Oxygen_School_temoignent`
+--
+
+CREATE TABLE `Student_Reviews` (
+  `id` INT NOT NULL,
+  `student_id` INT,
+  `testimonial` TEXT,
+  PRIMARY KEY (`id`),
+  FOREIGN KEY (`student_id`) REFERENCES `Student`(`id`)
+) ENGINE=InnoDB DEFAULT CHARSET=latin1;
```
Comment (Benja-san) [inline database.sql:123]: CHARSET to UTF8 (generalCI or AI CI)

File: database.sql [hunk 10]
```diff
@@ -61,3 +61,127 @@ ALTER TABLE `item`
...
+  FOREIGN KEY (`discipline_id`) REFERENCES `Discipline`(`id`)
+) ENGINE=InnoDB DEFAULT CHARSET=latin1;
+
+--
+-- Structure de la table `Discipline`
+--
+
+CREATE TABLE `Discipline` (
+  `id` INT NOT NULL,
+  `name` VARCHAR(100),
```
Comment (Benja-san) [inline database.sql:108]: can this field be null ?

File: database.sql [hunk 11]
```diff
@@ -61,3 +61,127 @@ ALTER TABLE `item`
...
+  `description` VARCHAR(150),
```
Comment (Benja-san) [inline database.sql:109]: can this field be null ? and are 150 characters for a description enough ?

**Review by Benja-san — CHANGES_REQUESTED** [review]

**Review by yazidsef — APPROVED** [review]

**Review by Benja-san — APPROVED** [review]

**Pull request merged**